SofaKernel/modules/SofaSimpleFem/src/SofaSimpleFem/HexahedronFEMForceField.inl

```diff
@@ -1191,6 +1191,7 @@ void HexahedronFEMForceField<DataTypes>::draw(const core::visual::VisualParams*
     if (!vparams->displayFlags().getShowForceFields()) return;
     if (!this->mstate) return;
     if (!f_drawing.getValue()) return;
+    if (!m_topology) return;
```

Comment:
Wont it be more consistent to rely on the componentState ? I saw that it is set to invalid if the topology is not found at init()

Reply:
I agree that we must rely on the component state. This is added in my last commit. But I also prefer an extra security and keep the pointer check.